pages/categoryPage/category.css

```diff
@@ -0,0 +1,913 @@
+    background-color: #222;
+    pointer-events: auto;
+    -webkit-appearance: none;
+    border-radius: 50%;
+}
+
+input[type='range']::-moz-range-thumb {
+    height: 13px;
+    width: 13px;
+    border: none;
+    background-color: #222;
+    pointer-events: auto;
+    -moz-appearance: none;
+    border-radius: 50%;
+}
+
+/* --- featuredItems --- */
+
+.featuredBlock {
+    padding-bottom: 2.5em;
+    margin-bottom: 2.5em;
+}
+
+.featuredBlock p {
+    font-family: 'Rubik', sans-serif;
+    font-weight: 500;
+    font-size: 1.2em;
+    color: #222;
+    margin: 0;
+}
+
+.featuredItemCard {
+    display: flex;
+    justify-content: space-between;
+    border-bottom: solid 1px #eff1f3;
+    padding: 1.5em 0;
+    height: 30%;
+}
+
+.featuredItemPhoto {
+    width: 25%;
+    height: 25%;
+    object-fit: cover;
+}
+
+.featuredItemInfo {
+    width: 67%;
+    display: flex;
+    flex-direction: column;
+}
+
+.featuredItemInfo .featuredItemPrice,
+.featuredItemInfo .featuredItemName {
+    font-family: 'Rubik', sans-serif;
+    font-weight: 500;
+    font-size: 0.825em;
+    color: #aaa;
+}
+
+.featuredItemInfo .featuredItemName {
+    color: #222;
+}
+
+.ratingIcons {
+    font-size: 0.725em;
+}
+
+.ratingIcons,
+.featuredItemPrice,
+.featuredItemName {
+    flex-grow: 1;
+}
+
+/*--------------- End of filtersSection ---------------*/
+
+/*--------------- productsSection ---------------*/
+
+.productsBlock {
+    display: flex;
+    justify-content: space-between;
+    margin: auto;
+    width: 60%;
+    padding: 4em 0;
+    gap: 5%;
+}
+
+.productsContainer {
+    width: 70%;
+    display: flex;
+    flex-direction: column;
+}
+
+.optionsContainer {
+    display: flex;
+    justify-content: space-between;
+}
+
+.filterButton {
+    display: none;
+    color: #666;
+    font-family: 'Rubik', serif;
+    font-size: 0.85em;
+    border: 2px solid #ddd;
+    text-align: center;
+}
+
+.sorting {
+    display: flex;
+    max-width: 100%;
+    gap: 2em;
+}
+
+.sortSelector {
+    color: #666;
+    font-family: 'Rubik', sans-serif;
+    padding: 1em 3em 1em 1.5em;
+    border: 2px solid #ddd;
+    border-radius: 0;
+    -webkit-appearance: none !important;
+    -moz-appearance: none !important;
+    max-width: 100%;
+    background-image: url("../../assets/icons/angle-down.svg");
+    background-size: 7%;
+    background-position: 90%;
+    background-repeat: no-repeat;
+}
+
+.sortSelector:focus {
+    outline: none;
+    border: 2px solid #222;
+    border-radius: 0;
+}
+
+.productsView {
+    display: flex;
+    justify-content: center;
+    max-width: 100%;
+}
+
+.iconView {
+    border: 2px solid #ddd;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: -2px;
+}
+
+.iconView img {
+    height: 40%;
+    padding: 0.8em;
+}
+
+.productCountInfo {
+    display: flex;
+    margin-left: 2em;
+}
+
+.productCountInfo p {
+    font-family: 'Rubik', sans-serif;
+    font-size: 0.9em;
+    white-space: nowrap;
```

Comment:
using white-space: nowrap; this is not a best practice 
In this case, if we have a long category name, it will not fit on the page